README.md

```diff
@@ -7,13 +7,13 @@ cd coupled-workflow
 git checkout feature/coupled-crow
 git submodule update --init --recursive     #Update submodules 
 cd sorc
-sh checkout.sh -c                           #Check out forecast model with COUPLED=YES
+sh checkout.sh -a                           #Check out forecast model with active aerosols
```

Comment:
Unless all the coupled model development is leaving the ufs-weather-model and moving to be used to the aerosol model, can the README please be modified so that this shows options for "coupled" or "aerosol" instead of aerosol only?